src/rgw/driver/sfs/sfs_gc.cc

```diff
@@ -44,31 +46,37 @@ SFSGC::~SFSGC() {
 int SFSGC::process() {
   // This is the method that does the garbage collection.
   initial_process_time = ceph_clock_now();
+  perfcounter->inc(l_rgw_sfs_gc_count);
 
   // start by deleting possible pending objects data in the filesystem
   // this could be stopped in a previous execution due to max exec time elapsed
   auto time_to_process_more = delete_pending_objects_data();
   if (!time_to_process_more) {
+    perfcounter->set(l_rgw_sfs_gc_process_exit, 1);
     return 0;
   }
   // now delete possible pending multiparts data
   time_to_process_more = delete_pending_multiparts_data();
   if (!time_to_process_more) {
+    perfcounter->set(l_rgw_sfs_gc_process_exit, 2);
     return 0;
   }
   // process deleted buckets
   time_to_process_more = process_deleted_buckets();
   if (!time_to_process_more) {
+    perfcounter->set(l_rgw_sfs_gc_process_exit, 3);
     return 0;
   }
   // process deleted objects
   time_to_process_more = process_deleted_objects();
   if (!time_to_process_more) {
+    perfcounter->set(l_rgw_sfs_gc_process_exit, 4);
```

Comment:
I think we can name those numbers... for a better reference.

Reply:
Agree. The gauge metric will still be a number (prometheus only does numbers), but I'll try to get constants and constant / number mapping info into the help text.